engine-vuex/src/store.ts

```diff
@@ -279,6 +280,60 @@ export class WWTEngineVuexModule extends VuexModule implements WWTEngineVuexStat
     }
   }
 
+  get findRADecForScreenPoint() {
+    return function (pt: { x: number; y: number }): { ra: number; dec: number } {
+      if (Vue.$wwt.inst === null)
+        throw new Error('cannot findRADecForScreenPoint without linking to WWTInstance');
+      const coords = Vue.$wwt.inst.ctl.getCoordinatesForScreenPoint(pt.x, pt.y);
+      return { ra: (15 * coords.x + 720) % 360, dec : coords.y };
```

Comment:
Yeah, this will make sure that the RA value is positive. I actually stole this from the [FinderScope](https://github.com/WorldWideTelescope/wwt-web-client/blob/master/factories/FinderScope.js#L27) in the web client.